source/bringauto/modules/ModuleHandler.cpp

```diff
@@ -131,14 +118,13 @@ void ModuleHandler::handleDisconnect(const structures::DeviceIdentification& dev
 void ModuleHandler::sendAggregatedStatus(const structures::DeviceIdentification &deviceId, const ip::Device &device,
 										 bool disconnected) {
 	auto &statusAggregator = moduleLibrary_.statusAggregators.at(deviceId.getModule());
```

Comment:
How do we know the given status aggregator is in the map struct?

Reply:
This function is being called only in handleDisconnect and handleStatus.
Both of those functions already check if that aggregator exists.